docs/2notebook/Example_6_Chinese Attack.ipynb

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
I think this notebook provides a really good illustration of using the Chinese attack. Providing more explanations and fixing up the formats would definitely make this better. For example, in the last section, there is no need to import `SwapRandomCharacterDeletion, WordSwapQWERTY, CompositeTransformation` since they are not used, which may confuse new users.

Reply:
Agreed